backend/pyproject.toml

```diff
@@ -42,8 +42,6 @@ include = ["zeno_backend"]
 
 [tool.ruff]
 select = ["E", "F", "I", "PD", "N", "D", "T20"]
-# Ignore conflicting specs between linter and formatter. Formatter will catch line length.
-ignore = ["D206", "E501"]
```

Comment:
This has to be removed otherwise the CI does not catch things that should be caught (e.g. long lines)